### PR TITLE
Fix the `tetra` CLI install directions for Windows

### PR DESCRIPTION
**Pull request by spkane**
Fix `tetra` CLI install directions for Windows

Thanks for contributing! Please ensure your pull request adheres to the following guidelines:

- [x] All commits contain a well written commit message and are signed-off (see [Submitting a pull request](https://tetragon.io/docs/contribution-guide/submitting-a-pull-request/)).
- [x] All code is covered by unit and/or end-to-end tests where feasible.
- [x] All generated files are updated if needed (see [Making changes](https://tetragon.io/docs/contribution-guide/making-changes/)).
- [x] Provide a title or release-note blurb suitable for the release notes (see [guidelines](https://tetragon.io/docs/contribution-guide/docs/release-notes/#release-note-blurb-in-pr)).
- [x] Update documentation and write an upgrade note if needed (see [guidelines](https://tetragon.io/docs/contribution-guide/docs/release-notes/#upgrade-notes)).
- [x] Are you a user of Tetragon? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md) in the Cilium repository.

- Fix the `tetra` CLI install directions for Windows

### CODE REVIEW

File: docs/content/en/docs/installation/tetra-cli.md [hunk 1]
```diff
@@ -70,14 +70,14 @@ sudo mv tetra /usr/local/bin
 {{< /tab >}}
 
 {{< tab header="Windows amd64" >}}
-curl -L https://github.com/cilium/tetragon/releases/latest/download/tetra-windows-amd64.tar.gz
-tar -xz tetra-windows-amd64.tar.gz
+curl -L https://github.com/cilium/tetragon/releases/latest/download/tetra-windows-amd64.tar.gz -O
+tar -xzf tetra-windows-amd64.tar.gz
```
Comment (michi-covalent) [inline docs/content/en/docs/installation/tetra-cli.md:74]: minor nit maybe better to put the flag in front of the argument:
```suggestion
curl -LO https://github.com/cilium/tetragon/releases/latest/download/tetra-windows-amd64.tar.gz
tar -xzf tetra-windows-amd64.tar.gz
```

File: docs/content/en/docs/installation/tetra-cli.md [hunk 2]
```diff
@@ -70,14 +70,14 @@ sudo mv tetra /usr/local/bin
...
 # move the binary in a directory in your PATH
 {{< /tab >}}
 
 {{< tab header="Windows arm64" >}}
-curl -L https://github.com/cilium/tetragon/releases/latest/download/tetra-windows-arm64.tar.gz
-tar -xz tetra-windows-arm64.tar.gz
+curl -L https://github.com/cilium/tetragon/releases/latest/download/tetra-windows-arm64.tar.gz -O
```
Comment (michi-covalent) [inline docs/content/en/docs/installation/tetra-cli.md:79]: same as above
```suggestion
curl -LO https://github.com/cilium/tetragon/releases/latest/download/tetra-windows-arm64.tar.gz
```

**Review by michi-covalent — APPROVED** [review]

**Review by mtardy — COMMENTED** [review]
Hey thanks! Yep agree with Michi, if you can group those args and eventually squash your two commits, that would be perfect :)

**Review by mtardy — APPROVED** [review]
Thanks!!

**Comment by spkane** [comment]
@mtardy and @michi-covalent - Changes made. This should be good to go now.

**Comment by spkane** [comment]
@mtardy I can't merge this personally, so someone else will have to take care of that.

**Comment by mtardy** [comment]
> @mtardy I can't merge this personally, so someone else will have to take care of that.

yeah I know no worries :)

**Pull request merged**